modules/floating-buttons/module.php

```diff
@@ -73,6 +73,33 @@ private function register_admin_menu_legacy( Admin_Menu_Manager $admin_menu ) {
 	public function __construct() {
 		parent::__construct();
 
+		if ( isset( $_GET['playwright_testing'] ) ) {
```

Comment:
@louiswol94 should I always register the meta? because without registering the meta I can't create the post through Rest endpoints, unless I create a custom endpoint

Reply:
Can you try `meta_input`?

Reply:
@louiswol94 with meta_input the meta is ignored, it doesn't give an error but the meta is ignored and post is not correctr. from my testing the only way I got this to work is by using the refactored create function. If you want I can remove the extra function and modify the old one, checking that everything works. 
Regarding registering the meta, I couldn't get this to work without registering the meta. 
Could we add this during the test:setup:playwright phase somehow?

Reply:
@nicoladj77 Sorry for the back and forth on this. Maybe we can use `wp import`?

https://github.com/elementor/elementor-pro/blob/d2fdd658f07ceabd2b48b06edae7e20e3ceeb9e8/tests/wp-env/config/import-sample-data.sh#L3

Reply:
@louiswol94 yes I can try to use that, should work.